state/state.go

```diff
@@ -329,5 +337,8 @@ func MakeGenesisState(genDoc *types.GenesisDoc) (State, error) {
 		LastHeightConsensusParamsChanged: genDoc.InitialHeight,
 
 		AppHash: genDoc.AppHash,
+
+		// NextBlockDelay is set to 0 because the genesis block is committed.
+		NextBlockDelay: 0,
```

Comment:
NOTE that previously, we've been waiting for 1s even for the genesis block, which doesn't make sense since validators don't vote for it.

Reply:
Just has a quick chat. We agreed:

* This hunk is correct: it doesn't make sense to put something different from 0 here
* If the user starts the chain with a (stale) value of (now deprecated) `timeout_commit` in `config.toml` bigger than zero, if value will override the here for the first block of the chain. But that's up to the user to fix, and anyway not a bug deal.

I'm happy we now have a common understanding. I think we can resolve this comment thread